packages/core/src/docs/typography.md

```diff
@@ -83,6 +83,9 @@ that contains text.
 
 @## Internationalization
 
+I18n in Blueprint is straightforward. React components expose props for customizing any strings;
+use the library of your choice for managing internationalized strings.
```

Comment:
wow that is great! do you want me to include a link to that here? seems somewhat tangential.